posts/2022-03-18-BillyDawson_MeetTheTeam.adoc

```diff
@@ -0,0 +1,167 @@
+
+At the start of 2021 my team took on WebSphere Developer Tools (WDT). The previous team mostly moved to Red Hat and unfortunately we were left with a product with significant technical debt, no handover and a problematic last release, with the aim of rapidly educating ourselves to return to the previous quarterly release cadence. It was an ambitious task and all the learning I have undertaken in prior years really helped to quickly ensure this target could be met. I led the releases for the first half of 2021 which was my first opportunity to properly dip my toes into project management. This project has given me open source exposure on two fronts - committing to our open source repository, OLT and given the plugin is built for Eclipse this has also allowed me to gain exposure to the Eclipse community. Our very first task was to resolve a customer issue within the OLT project and on the Eclipse front I was most recently liaising with the owner of a sub project that is a prerequisite for our product.
+
+There’s a wealth of additional activities one can participate in at IBM. I am the Hursley focal for developer advocacy which involves me planning and organising events, most frequently to reach out to UK universities. This has also given me several opportunities to present both to universities and to our customers. There have been several areas of technology I was particularly interested in - namely Quantum, AI and hacking - and despite not being part of my typical team work I was able to attend education sessions on Quantum and on hacking as well as an entire Coursera course on AI, which led to me being part of an AI work group to apply this technology to help automate workloads within the department. I attended presentation workshops to improve my confidence and communication skills and Customer Advisory Board calls to gain a greater insight into the types of customer interactions we typically see and the style in which we present to customers. I am also part of a patent group which has achieved a defensive publication. A really fun event I did prior to COVID was attending the local Marwell zoo for a volunteering day where we helped replant several hazel trees into the marabou stork enclosure, followed by some mulching and pruning, with the aim of providing privacy for the existing stork and its newly acquired male companion in the hope that they would mate. Fortunately this volunteering event will resume in 2022! Moreover, I attended the Call for Code hackathon and the Virgin Global Challenge – which saw our team competing, with provided fitness trackers, in a virtual race around the world.
```

Comment:
There have been several areas of technology I was particularly interested in - namely Quantum, AI and hacking. Despite not being part of my typical teamwork I was able to attend education sessions on Quantum and on hacking as well as an entire Coursera course on AI, which led to me being part of an AI work group to apply this technology to help automate workloads within the department.